pkg/gnmi-client-gen/generator_test.go

```diff
@@ -7,202 +7,261 @@
 package gnmi_client_gen
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	testdevice "github.com/onosproject/config-models/models/testdevice-1.0.x/api"
```

Comment:
this creates a looped dependency that could prevent further updates of YGOT

You could just import the []byte from generated.go like I did in `config-models/pkg/path/extract_test.go`

Reply:
I can remove this test altogether, I just needed it to extract the schema as it was helpful to see the entire structure while designing the templates, but it's not required and/or actually testing anything.

Just curious, why would it prevent updates to YGOT?

Reply:
in the last upgrade the schema []byte was changed by the YGOT upgrade. So if that was to happen again the testdevice, would have to be upgraded before config-models could be updated, and vice versa, creating a deadlock situation

Reply:
I think they could be upgraded at the same time as they are in the same package, but let me remove that tests as it's not actually a test anyway